File: DiagramasUML_Proy2/DiagramasUML_Proy2/Persona.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DiagramasUML_Proy2
{
    class Persona
    {
        private string name;
        private int age;

        public string getName()
        {
            return name;
        }
        public void setName(string name)
        {
            this.name = name;
        }
        public int getAge()
        {
            return age;
        }
        public void setAge(int age)
        {
            this.age = age;
        }

        public string toString()
        {
            return "Persona [name=" + name + ", age=" + age + "]";
        }
    }
}

File: Proyecto2_1096917/Proyecto2_1096917/Persona.cpp
#include "pch.h"
#include "Persona.h"
#include <iostream>

using namespace std;
Persona::Persona()
{
}

char* Persona::getName()
{
	return name;
}
void Persona::setName(char pName[])
{
	name = pName;
}
int Persona::getAge()
{
	return age;
}
void Persona::setAge(int pAge)
{
	age = pAge;
}

/*void Persona::toString()
{	
	cout << "[Name]: ";
	for (size_t i = 0; i < sizeof(name); i++)
	{
		cout << name[i];
	}
	cout << ", [Age]: " + age;
}*/

Persona::~Persona()
{
}
